orebfuscator-plugin/src/main/java/net/imprex/orebfuscator/config/OrebfuscatorProximityConfig.java

```diff
@@ -199,8 +203,13 @@ public Matrix4f frustumCullingProjectionMatrix() {
 	}
 
 	@Override
-	public boolean useRayCastCheck() {
-		return this.useRayCastCheck;
+	public boolean rayCastCheckEnabled() {
```

Comment:
Can't compile because you didn't push the changes you made to the [`ProximityConfig`](https://github.com/Imprex-Development/orebfuscator/blob/master/orebfuscator-common/src/main/java/net/imprex/orebfuscator/config/ProximityConfig.java) file

Reply:
Sorry. Because my IDE generates a lot of other files. I accidentally deleted this change when I was undoing it.